lib/usd/ui/USDImportDialogCmd.cpp

```diff
@@ -202,7 +221,7 @@ MStatus USDImportDialogCmd::doIt(const MArgList& args)
 			// Creating the View can pause Maya, usually only briefly but it's noticable, so we'll toggle the wait cursor to show that it's working.
 			QApplication::setOverrideCursor(QCursor(Qt::WaitCursor));
 
-			std::unique_ptr<IUSDImportView> usdImportDialog(new USDImportDialog(assetPath.asChar(), &importData, usdQtUtil, MQtUtil::mainWindow()));
+			std::unique_ptr<USDImportDialog> usdImportDialog(new USDImportDialog(assetPath.asChar(), &importData, usdQtUtil, MQtUtil::mainWindow()));
```

Comment:
Changing to use the derived class so I can call the functions that are not part of the base interface.  We can probably get rid of that interface class if we wanted to, but I have not bothered here.